packages/snaps-sdk/src/jsx/validation.ts

```diff
@@ -161,7 +161,7 @@ export const OptionStruct: Describe<OptionElement> = element('Option', {
 export const DropdownStruct: Describe<DropdownElement> = element('Dropdown', {
   name: string(),
   value: optional(string()),
-  children: maybeArray(OptionStruct),
+  children: maybeArray(nullUnion([OptionStruct, boolean()])),
```

Comment:
Realizing that this means that null isn't allowed in Dropdowns currently.

Would it be easier to have a `jsxChildren` struct that takes a list of structs and combines that with `boolean()`, `null` etc?

Reply:
Sure, that does sound easier.